boards/common/nrf52/Makefile.include

```diff
@@ -8,13 +8,24 @@ ifeq (bmp,$(PROGRAMMER))
   PORT_DARWIN ?= $(wordlist 2, 2, $(sort $(wildcard /dev/tty.usbmodem*)))
 endif
 
-# keep name of `JLINK` in sync with script jlink.sh in $(RIOTTOOLS)/jlink
-JLINK ?= JLinkExe
-ifneq (,$(command -v $(JLINK)))
+ifneq (,$(filter $(BOARD),ruuvitag thingy52))
+  # openocd doesn't fully work with ruuvitag and thingy52
   PROGRAMMER ?= jlink
+  ifeq (openocd,$(PROGRAMMER))
+    $(error Cannot use OpenOCD with $(BOARD) board)
```

Comment:
Should we really prohibit the use of OpenOCD? What if it gets fixed in a newer version?

Reply:
To be honest, I almost certain that it would work just fine with OpenOCD. I think one colleague of mine has a Thingy52 on his desk, so I will test that later.

I also cannot really understand why `$(error)` is used here when there already is a check against `PROGRAMMERS_SUPPORTED`.

But let's keep the existing behavior for now and just get Murdock passing.